### PR TITLE
v5.35 Notice

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: notices.json [hunk 1]
```diff
@@ -37,21 +37,21 @@
...
+      "serverVersion": ["<5.35"],
       "instanceType": "onprem",
-      "displayDate": ">= 2021-04-21T00:00:00Z"
+      "displayDate": ">= 2021-05-21T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
         "title": "New server version available",
-        "description": "The new feature release offers enhanced automation for incident collaboration, new plugins, and more. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
+        "description": "The new feature release offers shared channels, add/remove ad hoc tasks for incident collaboration playbooks, file search, and more. [Please also see the important upgrade notes](https://docs.mattermost.com/administration/important-upgrade-notes.html).",
```
Comment (esethna) [inline notices.json:51]: @amyblais wondering if we can summarize the improvements better? Perhaps:

"Stakeholder overview and ad hoc tasks for incident collaboration" 0/5

**Review by esethna — CHANGES_REQUESTED** [review]

**Review by esethna — APPROVED** [review]

**Review by ogi-m — APPROVED** [review]

**Comment by amyblais** [comment]
@cpanato I'm unable to merge this because of the incomplete git.internal check. Wondering if you can help?

**Comment by cpanato** [comment]
i will fix that later and give you the access as well

merging

**Comment by cpanato** [comment]
@amyblais can you please make a ticket for that?

**Pull request merged**